extension/src/popup/components/signTransaction/Transaction.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
```

Comment:
a lot of the work in this PR is splitting off a reusable component from `SignTransaction`